sky/global_user_state.py

```diff
@@ -125,15 +126,12 @@ class ClusterStatus(enum.Enum):
     # Stopped.  This means a `sky stop` call has previously succeeded.
     STOPPED = 'STOPPED'
 
-    def colored_str(self):
-        color = _STATUS_TO_COLOR[self]
-        return f'{color}{self.value}{colorama.Style.RESET_ALL}'
-
 
-_STATUS_TO_COLOR = {
-    ClusterStatus.INIT: colorama.Fore.BLUE,
-    ClusterStatus.UP: colorama.Fore.GREEN,
-    ClusterStatus.STOPPED: colorama.Fore.YELLOW,
+STATUS_TO_COLOR = {
+    ClusterStatus.INIT.value: colorama.Fore.BLUE,
+    ClusterStatus.UP.value: colorama.Fore.GREEN,
+    ClusterStatus.STOPPED.value: colorama.Fore.YELLOW,
+    'TERMINATED': colorama.Fore.BLACK,  # sky down
```

Comment:
I am sorry, but after looking at the new changes, I found it might be better to revert this by adding the `TERMINATED` status to the `ClusterStatus`. Can we revert to your original changes for the `ClusterStatus` and move that `colored_str` back in the the `ClusterStatus` class?